Automattic/wp-calypso
# Home: Mobile setup tasks reference the Jetpack app

#### Proposed Changes

Following the plans to [refocus the WordPress apps on core features](https://make.wordpress.org/mobile/2022/07/27/refocusing-the-wordpress-app-on-core-features/), we plan to move features that are specific to WordPress.com to the Jetpack app. As such, we will eventually replace all mentions of the WordPress app in Calypso with Jetpack.

The following changes only apply to mobile device; larger device continue to see the current experience. A PR for larger devices will arrive later. 

* Add a Jetpack app overlay to _Home_. 
* Remove one of two app CTAs found on the home page for mobile web browsers. 
* Update copy/links for the retained app CTA to focus on the Jetpack app. 


<details><summary>UX Video</summary>

https://user-images.githubusercontent.com/438664/191395653-850b179d-cb92-4594-9331-ea218d5f8f5f.mov

</details>

<details><summary>Screenshots</summary>

| Before (Mobile) | After (Mobile) |
| - | - |
| ![home-mobile-before](https://user-images.githubusercontent.com/438664/191369006-258fc510-f04f-4983-bde5-c3643a7fc975.png) | ![home-mobile-after](https://user-images.githubusercontent.com/438664/191395330-f692ee48-e71f-42d8-a4dc-845d8e9016b1.png) |
| ![home-mobile-before-2](https://user-images.githubusercontent.com/438664/191369005-fa77d4aa-82c0-4876-86f7-ec676450b82e.png) |  <img width="376" alt="home-mobile-after-2" src="https://user-images.githubusercontent.com/438664/192872336-74a79556-2087-40d7-a0cf-5a2134ea8e8c.png"> |
| | ![home-mobile-overlay](https://user-images.githubusercontent.com/438664/191369007-68773a0c-5c1f-498a-adfc-93b1907220c8.png) |

| Before (Desktop) | After (Desktop) |
| - | - |
| ![home-desktop-before](https://user-images.githubusercontent.com/438664/191369001-c589c9b2-adac-4e80-b544-38ff729dcf29.png) | ![home-desktop-after](https://user-images.githubusercontent.com/438664/191392235-3e22d8f7-f51a-4865-8ed8-7f25dde3ba7b.png) |

</details>

#### Testing Instructions

* Verify the _Home_ overlay displays for mobile web browsers only and links to the Jetpack app.<sup>1</sup>
* Verify the only one app CTA is displayed in mobile web browsers and links to the Jetpack app. 
* Verify the app CTA on non-mobile web browsers remains the same as production and links to the WordPress mobile app. 

<sup>1. If the banner has already been dismissed, you can run `dispatch( { type: 'PREFERENCES_SET', key: 'appBannerDismissTimes', value: null } );` in the browser console. In order to run this in the console, you must use a local development server or append `?debug` to the URL.</sup>

#### Pre-merge Checklist
